packages/taquito-michel-codec/src/michelson-types.ts

```diff
@@ -225,6 +226,7 @@ export interface MichelsonTypeContract<T extends MichelsonType> extends TypeX<'c
 export interface MichelsonTypeOr<T extends [MichelsonType, MichelsonType]> extends TypeX<'or', T> {}
 export interface MichelsonTypeLambda<Arg extends MichelsonType, Ret extends MichelsonType>
   extends TypeX<'lambda', [Arg, Ret]> {}
+
```

Comment:
There is a `data` section in this file, I think we need `Lambda_rec` in `MichelsonDataID` at least (maybe in other places too).

Reply:
@e-asphyx can you validate?